packages/radio-button/src/presenters/RadioButtonPresenter.js

```diff
@@ -60,56 +56,52 @@ export default class RadioButtonPresenter extends Component {
     value: "value"
   };
 
-  id = generateId("radio-button");
-
   render() {
     const {
+      id,
```

Comment:
also should `id` be passed in as part of `...otherProps`

Reply:
👍